translator/var.go

```diff
@@ -62,13 +55,38 @@ func RecordAssignVarType(fset *token.FileSet, ret *ast.AssignStmt) {
 	fmt.Println("---------------------")
 }
 
+func getBasicLitType(expr *ast.BasicLit) string {
+	switch expr.Kind {
+	case token.INT:
+		return "int"
+	case token.FLOAT:
+		return "float64"
+	case token.STRING:
+		return "string"
+	case token.CHAR:
+		return "char"
+	}
+	return ""
+}
+
 func RecordDeclVarType(fset *token.FileSet, ret *ast.ValueSpec) {
 	fmt.Println("---------------------")
-	for _, declVar := range ret.Names {
-		declVarType := reflectType(fset, ret.Type)
-		fmt.Println("-- declVar ", declVar, " declare type ...... ", declVarType)
-		variableType[declVar.Name] = declVarType
-		fmt.Println("[variableType] is ", variableType)
+	for i, declVar := range ret.Names {
+		if lenOfValues := len(ret.Values); lenOfValues == 0 {
```

Comment:
The syntax is strange here. How about just use ` len(ret.Values)`.

I think it is expressive as `lenOfValues`.